sale_input_barcode_gs1/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2022 Akretion
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Sale Input Barcode GS1",
+    "version": "14.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/stock-logistics-barcode",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "summary": "Add Sale line with barcode using GS1 barcodes",
+    "maintainers": ["bealdav"],
+    "depends": [
+        "base_gs1_barcode",
+        "sale_input_barcode",
+    ],
+    "data": [],
+    "demo": [],
+    "auto_install": True,
```

Comment:
I'm not sure it's necessary. I suppose it's better to recommend to install it in readme